src/botPage/view/TradeInfoPanel/TradeTable.js

```diff
@@ -126,6 +139,38 @@ export default class TradeTable extends Component {
         }
         return this.state[accountID];
     }
+    updateTable(info) {
+        if (!info) {
+            return;
+        }
+        const buyDate = new Date(info.date_start * 1000);
+        const timestamp = `${buyDate.toISOString().split('T')[0]} ${buyDate.toTimeString().slice(0, 8)} ${
```

Comment:
Since we're using this in both the `bot.contract` event (see `componentDidMount()`) and here, we could create a common function for parsing the timestamp.